src/scripts/install.sh

```diff
@@ -33,7 +33,13 @@ parsedrelease=$(echo "$PARAM_RELEASE" | tr '[:upper:]' '[:lower:]')
 if [[ $parsedrelease = "latest" ]]; then
     mpmrelease=$(curl https://ssd.mathworks.com/supportfiles/ci/matlab-release/v0/latest)
 else
-    mpmrelease="${PARAM_RELEASE}"
+    mpmrelease="${parsedrelease}"
```

Comment:
What was the rationality before of using the users input vs the formatted lowercase version?

Reply:
We probably should've always done it like this imo, but mpm isn't case sensitive (except for one bug that was gecked) so it didn't cause any problems. We need it to be lowercase now though for the string comparison to work.

Reply:
Gotcha!